application-templates/javascript/assets/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "assets",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc && vite build",
+    "preview": "vite preview"
+  },
+  "devDependencies": {
+    "typescript": "^5.2.2",
```

Comment:
👀

Reply:
If you never installed typescript in a javascript project, you're losing on something 😂